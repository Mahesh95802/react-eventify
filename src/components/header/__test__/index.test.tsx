import React from 'react';
import { render, screen } from '@testing-library/react';
import Header from '..';

test('renders learn react link', () => {
  const { asFragment } = render(<Header />);
  expect(asFragment()).toMatchSnapshot();
});
