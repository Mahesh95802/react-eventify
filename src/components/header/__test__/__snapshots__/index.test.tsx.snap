// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly 1`] = `
<div>
  <header
    class="header basic-padding"
  >
    <p>
      My 
      <span
        style="font-weight: bolder;"
      >
        Record
      </span>
       Shelf
    </p>
  </header>
</div>
`;
